routes/index.js

```diff
@@ -23,5 +23,6 @@ router.all('/estate/inskrivning', getInskrivning);
 router.all('/proxy', proxy);
 router.all('/lmproxy/*', lmProxy);
 router.all('/lmproxy-ver/*', lmProxyVer);
+router.all('/*', getAkt);
```

Comment:
Maybe aktdirekt should have its own route? Something like:
``` 
router.all('/aktdirekt/*', getAkt);
```

Reply:
I agree, this would seem more logical. There are add-on modules to http-proxy that handle this but I preffered to use this route before adding on dependencies. If this routing is causing problems we can change it.

Reply:
Could you clarify a bit @mulfvik? Why wouldn't it work writing it like @daRasmussen suggested?

Reply:
Ok, we could use an add-on module to rewrite the routing, e.g. [http-proxy-rules](https://www.npmjs.com/package/http-proxy-rules), but I didnt find any reason for that when we still get the Origo server startpage and Im guessing upcoming further development will not use that route for anything except that. 
The http-proxy **target** "excludes" any paths so thats why `const targetPath = "/distribution/produkter/aktdirekt/v3.0";` is included as part of the proxy request path. By using `router.all('/aktdirekt/*', getAkt);` we break the processed request string. 
So... it would work as @daRasmussen suggested but if its worth it, I dont know. 
The performance is ok, for me, as is.
Does this answer your question @markus659, what do you think about this ?

Reply:
@mulfvik what I was referring to is: 

Detta är en tjänst för att integrera åtkomst till *vissa* av Lantmäteriets arkiv i egna system.

[Akt Direkt - Teknisk beskrivning](https://www.lantmateriet.se/contentassets/a3c04499c4c04eb3a524a779e4ab3f80/tb_akt-direkt-v1.0.pdf)

Reply:
Hi @daRasmussen, aha, ok. So there are other servcies that serve other parts of their archives?

Reply:
@daRasmussen As far as I know this is the only service used for reading documents from Lantmäteriets archives. If the document cant be found with this service one need to contact geodatasupport@lm.se and they will help one further.

Reply:
@mulfvik Reading some other type of data (not documents from LM) should always be the primary route for origo-server? Akt-direkt should be the primary path :dancing_men:  It's cool but maybe it should be specified in the README?

Reply:
Thank you for clarifying @daRasmussen 🥇